src/kimmdy/plugins.py

```diff
@@ -98,14 +98,18 @@ class Parameterizer(ABC):
     type_scheme = dict()
 
     @abstractmethod
-    def parameterize_topology(self, current_topology: Topology) -> Topology:
+    def parameterize_topology(
+        self, current_topology: Topology, focus_nrs: dict[str, set[str]] = {}
```

Comment:
same problem as in https://github.com/hits-mbm-dev/kimmdy-grappa/pull/16/files/3583f7126275cde6f2506c790dbd144b7f347158
(mutable default values)